.github/workflows/release_gem.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    tags:
+      - 'v*'
```

Comment:
the tags in this repo do not use a `v` prefix: https://github.com/theforeman/ldap_fluff/tags